pkg/scheduler/scheduler_tas_test.go

```diff
@@ -147,6 +147,42 @@ func TestScheduleForTAS(t *testing.T) {
 			Obj(),
 	}
 
+	//    r1
+	//  /  |  \
+	// x1  x2  x3
+	partiallyEmptyNodes := []corev1.Node{
```

Comment:
I think `partiallyEmptyNodes` does not really capture the intention well, which is to say: "nodes with at least one instance which has no free capacity". Such nodes I would say are rather "full" or "exhausted" rather than "empty". 

Maybe "nodesWithExhaustedCapacity"?

Reply:
or `nodesWithZeroCapacityInstance`